schema/source_schema.json

```diff
@@ -18,6 +18,10 @@
   ],
   "additionalProperties": false,
   "properties": {
+    "countryCode": {
+      "type": "string",
+      "description": "Two letter country code"
+    },
```

Comment:
Country code was added to sources metadata to support future capabilities for automatic generation of visualizations such as those seen in the screenshot in this PR.